xml/System.Formats.Cbor/CborWriter.xml

```diff
@@ -299,14 +312,10 @@
         <summary>Writes the provided value as a tagged bignum encoding, as described in RFC7049 section 2.4.2.</summary>
         <remarks>To be added.</remarks>
         <exception cref="T:System.InvalidOperationException">Writing a new value exceeds the definite length of the parent data item.
-
--or-
-
-The major type of the encoded value is not permitted in the parent data item.
-
--or-
-
-The written data is not accepted under the current conformance mode.</exception>
+            -or-
```

Comment:
Looks like the formatting improvements weren't backported, so I think I should rerun without importing XML comments. @carlossanlop